src/apis/DataInputApi.ts

```diff
@@ -0,0 +1,17 @@
+
+import axios from 'axios';
+import { DataInputRequest, DataInputResponse } from '../types/Types';
+
+const BASE_URL = 'https://75iaqg7l2j.execute-api.ap-south-1.amazonaws.com/prod'; // Replace with your actual API base URL
+
+export const postDataInput = async (request: DataInputRequest): Promise<DataInputResponse> => {
```

Comment:
`DataInputRequest` should just be same as `User`

This is the input payload to DataInputApi:
```
{
  "userId": "asdfasdf",
  "firstName": "vaibhav",
  "lastName": "vasant",
  "email": "v12@gmail.com",
  "phoneNumber": "7633824597",
  "address": "None"
}
```